core/docker/bin/run-trino

```diff
@@ -2,9 +2,19 @@
 
 set -xeuo pipefail
 
+function graceful_shutdown()
+{
+    echo "Shutting down this worker gracefully"
+    curl -v -X PUT -d '"SHUTTING_DOWN"' -H "Content-type: application/json" http://localhost:8080/v1/info/state
```

Comment:
There should be timeouts defined here. What would be the typical expected value for a healthy cluster? Does it depend on the configuration, like some query timeouts?

How to handle errors, when this command would fail? I don't think `wait` has a timeout so we'd need to send a signal to the launcher.

Reply:
I added retry and timeout to curl in [d8aa476](https://github.com/trinodb/trino/pull/15180/commits/d8aa4765cec3a54c86202691b81d4b870e0bf942).